codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/protocols/HttpBoundProtocolGenerator.kt

```diff
@@ -62,10 +61,10 @@ class HttpBoundProtocolGenerator(
     HttpBoundProtocolTraitImplGenerator(codegenContext, protocol),
 )
 
-class HttpBoundProtocolTraitImplGenerator(
+open class HttpBoundProtocolTraitImplGenerator(
```

Comment:
I'm assuming it's `open` for the work in progress you mention in the overview?

Reply:
In `codegen-client`, one of the unit tests inherit and override one of the methods in this class. Previously, the unit tests just implemented the interface.